src/resolvers/Mutation/assignUserTag.ts

```diff
@@ -74,7 +74,7 @@ export const assignUserTag: MutationResolvers["assignUserTag"] = async (
   const requestUserBelongsToTagOrganization =
     requestUser.joinedOrganizations.some(
       (organization) =>
-        organization.toString() === tag!.organizationId.toString()
+        organization.toString() === tag.organizationId.toString()
```

Comment:
@KrutikaBhatt mongoose has inbuilt method of comparing mongodb objectIds. The method exists on every mongoose objectId. Comparision is done like this
`objectId1.equals(objectId2)`. Read more about it, and I'd say use it throughout the codebase whenever a comparision for mongodb objectIds is encountered.

Reply:
Yes sir, I have made these changes and tested them. Will push the changes